server/src/server.js

```diff
@@ -1112,7 +1142,9 @@ app.get("/api/fxa-oauth/confirm-login", function(req, res, next) {
               ua: req.headers["user-agent"],
             }).send();
           }
-          res.redirect("/settings");
+          // Redirect to endpoint
+          const pageUri = endpoint ? "/" + endpoint : "/";
```

Comment:
Would My Shots make more sense as the default?  I don't know what the new home page is going to have for content, but I think users want to see their shots after logging in.

Reply:
Redirecting to my shots for now until we get specific UX feedback on the flow